rj_gameplay/rj_gameplay/tactic/pass_tactic.py

```diff
@@ -50,19 +50,17 @@ def __call__(self,
             else:
                 return 1
 
-
-
 class Pass(tactic.ITactic):
     """
     A passing tactic which captures then passes the ball
     """
 
-    def __init__(self, target_point:np.ndarray):
+    def __init__(self, target_point:np.ndarray, passer_cost:role.CostFn, receiver_cost:role.CostFn):
         self.target_point = target_point
         self.pivot_kick = tactic.SkillEntry(pivot_kick.PivotKick(target_point = target_point, chip=False, kick_speed=4.0))
         self.receive = tactic.SkillEntry(receive.Receive())
```

Comment:
For some reason robot 7's receive is settling, but not collecting the ball. (If my understanding is correct, settling = making the ball's velocity go to zero, and receiving = actually putting the ball in mouth.) Is this receive skill updated with the recent changes from #1644 ?

Reply:
Or is it something conflicting with Robot 7's "move to corner" command right now?